engine/engine.go

```diff
@@ -211,6 +211,8 @@ func (e Engine) launchResolution(publicID string, async bool, sm *semaphore.Weig
 		return nil, nil
 	}
 
+	debugLogger = debugLogger.WithFields(logrus.Fields{"template_name": t.TemplateName, "task_id": t.PublicID})
```

Comment:
Should use the constants added in `pkg/metadata` package as field names in this file.